src/domains/sign/io_sign_support_func.tf

```diff
@@ -0,0 +1,186 @@
+      NODE_ENV                                        = "production"
+      CosmosDbConnectionString                        = module.cosmosdb_account.connection_strings[0]
+      CosmosDbIssuerDatabaseName                      = module.cosmosdb_sql_database_issuer.name
+      CosmosDbUserDatabaseName                        = module.cosmosdb_sql_database_user.name
+      PdvTokenizerApiBasePath                         = "https://api.uat.tokenizer.pdv.pagopa.it"
+      PdvTokenizerApiKey                              = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
+    }
+  }
+}
+
+module "io_sign_support_func" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v4.1.6"
+
+  name                = format("%s-support-func", local.project)
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+
+  health_check_path = "/api/v1/sign/support/info"
+
+  os_type          = "linux"
+  runtime_version  = "~4"
+  always_on        = true
+  linux_fx_version = "NODE|18"
+
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_tier                     = var.io_sign_support_func.sku_tier
+    sku_size                     = var.io_sign_support_func.sku_size
+    maximum_elastic_worker_count = 0
+  }
+
+  app_settings = local.io_sign_support_func.app_settings
+
+  subnet_id       = module.io_sign_snet.id
+  allowed_subnets = [module.io_sign_snet.id, data.azurerm_subnet.apim.id]
+
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+  system_identity_enabled                  = true
+
+  tags = var.tags
+}
+
+module "io_sign_support_func_staging_slot" {
+  count  = var.io_sign_support_func.sku_tier == "PremiumV3" ? 1 : 0
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app_slot?ref=v4.1.3"
+
+  name                = "staging"
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+  function_app_name   = module.io_sign_support_func.name
+  function_app_id     = module.io_sign_support_func.id
+  app_service_plan_id = module.io_sign_support_func.app_service_plan_id
+
+  health_check_path = "/api/v1/sign/support/info"
+
+  storage_account_name       = module.io_sign_support_func.storage_account.name
+  storage_account_access_key = module.io_sign_support_func.storage_account.primary_access_key
+
+  os_type                                  = "linux"
+  runtime_version                          = "~4"
+  always_on                                = true
+  linux_fx_version                         = "NODE|18"
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+
+  app_settings = local.io_sign_support_func.app_settings
+
+  subnet_id = module.io_sign_snet.id
```

Comment:
ditto.

The additional slot however has to be in the same subnet as the main function_App